lib/mix/tasks/phx.gen.html.ex

```diff
@@ -4,15 +4,16 @@ defmodule Mix.Tasks.Phx.Gen.Html do
   @moduledoc """
   Generates controller with view, templates, schema and context for an HTML resource.
 
-      mix phx.gen.html Accounts User users name:string age:integer
+      mix phx.gen.html Accounts User users name:string age:integer about:text
```

Comment:
I find that using a `textarea` for `text`-type fields is a positive change, but the current docs changes do not help convey that distinction. It is probably not worth documenting anyway, as people are free and encouraged to change the generated code.

IMHO I'd keep the example `mix` call shorter as it was.

Reply:
That's fair enough. :)